packages/ai/cypress/specs/AINoticeIndicator.cy.tsx

```diff
@@ -1,109 +1,162 @@
 import AINoticeIndicator from "../../src/AINoticeIndicator.js";
+import {
+	AI_NOTICE_INDICATOR_CLOSE_BUTTON_TEXT,
+	AI_NOTICE_INDICATOR_POPOVER_CONTENT,
+	AI_NOTICE_INDICATOR_ATTRIBUTIONTEXT,
+	AI_NOTICE_INDICATOR_VERIFICATIONTEXT
+} from "../../src/generated/i18n/i18n-defaults.js";
 
 describe("AI Notice Indicator", () => {
 	describe("Rendering and Interaction", () => {
 		beforeEach(() => {
 			cy.mount(<AINoticeIndicator></AINoticeIndicator>);
 		});
 
-		const attributionText = "Created with AI.";
-		const verificationText = "Verify results before use.";
-
 		it('Should render the component', () => {
 			cy.get('ui5-ai-notice-indicator').should('exist');
 		});
 
-		it("Should display closed responsive poppover", () => {
+		it("Should display closed responsive popover", () => {
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-responsive-popover]")
 				.should("exist");
-
-			cy.get("[ui5-ai-notice-indicator]")
-				.shadow()
-				.find("[ui5-responsive-popover]")
-				.should("have.prop", "open", false);
 		});
 
-		it("Should display correct properties for Default mode", () => {
+		it("Should display correct elements for Default mode", () => {
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.should('have.attr', 'icon', '')
+				.should('not.have.attr', 'icon');
+
+			cy.get("[ui5-ai-notice-indicator]")
+				.shadow()
+				.find("[ui5-label]")
 				.should("exist");
+		});
+
+		it("Should display correct elements for Emphasized mode", () => {
+			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Emphasized');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.contains(attributionText)
+				.should('have.attr', 'icon', 'ai')
 				.should("exist");
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-label]")
-				.contains(verificationText)
 				.should("exist");
 		});
 
-		it("Should display correct properties for Emphasized mode", () => {
-			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Emphasized');
+		it("Should display correct elements for Shortened mode", () => {
+			cy.get('ui5-ai-notice-indicator').invoke('attr', 'mode', 'Shortened');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.should('have.attr', 'icon', 'ai')
-				.should("exist");
+				.should('not.have.attr', 'icon');
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-link]")
-				.contains(attributionText)
 				.should("exist");
 
 			cy.get("[ui5-ai-notice-indicator]")
 				.shadow()
 				.find("[ui5-label]")
-				.contains(verificationText)
 				.should("exist");
```

Comment:
Shortened mode should not have a label present at all, just the Link. Update this test once you fix this in the renderer by removing this part of it where you check for the label